USAGE

```diff
@@ -15,13 +41,17 @@ DESCRIPTION
                                     │   ├── *fodf.nii.gz
                                     │   ├── *map_exclude.nii.gz (optional, pft)
                                     │   ├── *map_include.nii.gz (optional, pft)
+                                    │   ├── *0GenericAffine.mat (optional, transfo)
+                                    │   ├── *Inverse1Warp.nii.gz (optional, transfo)
```

Comment:
Is it okay if I simply say they should come from Tractoflow? 
Otherwise I have to explain it should be native BET T1w to BET fa and that the headers must match